pandas/core/generic.py

```diff
@@ -2920,6 +2920,9 @@ def interpolate(self, method='linear', axis=0, limit=None, inplace=False,
             * 1: fill row-by-row
         limit : int, default None.
             Maximum number of consecutive NaNs to fill.
+        limit_direction : {'forward', 'backward', 'both'}, defaults to 'forward'
+            If limit is specified, consecutive NaNs will be filled in this
+            direction.
```

Comment:
Can you add here `.. versionadded:: 0.17.0` ? (see here for how to format: https://github.com/pydata/pandas/blob/v0.16.2/pandas/core/frame.py#L1171)